### PR TITLE
Update import.rst

### PR DESCRIPTION
**Pull request by abdounikarim**
Hello,

I just add a configuration if we want to import all configuration of a directory, to avoid multiple imports lines.

### CODE REVIEW

File: service_container/import.rst [hunk 1]
```diff
@@ -122,6 +125,8 @@ a relative or absolute path to the imported file:
 
         return function(ContainerConfigurator $configurator) {
             $configurator->import('services/mailer.php');
+            //If you want to import a directory, use the next configuration
```
Comment (Matts) [inline service_container/import.rst]: missing space in front of the line.

File: service_container/import.rst [hunk 2]
```diff
@@ -80,7 +80,8 @@ a relative or absolute path to the imported file:
         # config/services.yaml
         imports:
             - { resource: services/mailer.yaml }
-
+            # If you want to import a directory, use the next configuration
+            - { resource: services/ }
```
Comment (Matts) [inline service_container/import.rst:84]: Tbh, it does look kind of weird with the unquoted trailing `/`, but guess this is how they implemented this functionality. So this is okay

**Review by Matts — CHANGES_REQUESTED** [review]
Hey @abdounikarim 
Besides a space missing on line 128 LGTM

File: service_container/import.rst (hunk 1 above)
Comment (abdounikarim) [reply service_container/import.rst]: done

**Review by Matts — APPROVED** [review]

File: service_container/import.rst [hunk 3]
```diff
@@ -80,7 +80,8 @@ a relative or absolute path to the imported file:
         # config/services.yaml
         imports:
             - { resource: services/mailer.yaml }
-
+            # If you want to import a directory, use the next configuration
```
Comment (OskarStark) [inline service_container/import.rst]:
```suggestion
            # If you want to import a whole directory:
```

Comment (OskarStark) [reply service_container/import.rst]: Same for the other

Comment (abdounikarim) [reply service_container/import.rst]: done

**Review by OskarStark — APPROVED** [review]

File: service_container/import.rst [hunk 4]
```diff
@@ -122,6 +125,8 @@ a relative or absolute path to the imported file:
 
         return function(ContainerConfigurator $configurator) {
             $configurator->import('services/mailer.php');
+            // If you want to import a directory, use the next configuration
```
Comment (OskarStark) [inline service_container/import.rst]:
```suggestion
            // If you want to import a whole directory:
```

**Comment by OskarStark** [comment]
Thank you Abdouni.

**Pull request merged**